src/app/backend/resource/cronjob/common_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cronjob_test
```

Comment:
package cronjob?

Reply:
https://blog.golang.org/examples

`_test` suffix is used for a proper black box tests. It is allowed by the compiler and because it is a different package usage of package private functions is not allowed. Only public functions can be executed and tested. This is ok.